site/config.toml

```diff
@@ -12,6 +12,4 @@ title = "StatusKit"
 # List of systems monitored in this status page.
 # You'll be able to change their status every time
 # you open or update an incident.
-# Example:
-# systems = ["API", "CDN", "DNS", "Site delivery"]
-systems = []
+systems = ["API", "CDN", "DNS", "Site delivery"]
```

Comment:
I think pre-filled is much nicer. The site builds and shows the status boxes instead of nothing. I don't use the deploy to netlify flow, so that is untested from me. I'll follow your lead on this one.

Reply:
Yeah, I agree.

Since there's no longer an explicit example, let's add a comment to say,
```
# Replace these examples with your own system names.
```

Reply:
done